File: Linear Regression.R
#load train and test sets

X_train <- input_variables_values_training_datasets
y_train <- target_variables_values_training_datasets
x_test <- input_variables_values_test_datasets

x <- cbind(x_train, y_train)

#train the model using the traing sets and check result

m <- lm(y_train ~ ., data = x)
summary(m)

#predict

predicted <- predict(m, x_test)

File: Linear Regression.py
from sklearn import linear_model

# load train and test sets

x_train = input_variables_values_training_datasets

y_train = target_variables_values_training_datasets

x_test = input_variables_values_test_datasets

# create linear regression object

model = linear_model.LinearRegression()

# train the model using the training sets and check score

model.fit(x_train, y_train)
model.score(x_train, y_train)

# check coefficient and intercept

print('coefficient: n', model.coef_)
print('Intercept: n', model.intercept_)

# predict output

predicted = model.predict(x_test)